app/src/main/res/layout/content_list_fragment.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools">
+  <data>
+    <variable
+      name="contentFragment"
+      type="org.oppia.app.player.content.ContentListFragment"/>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.content.ContentViewModel"/>
+  </data>
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:context=".player.content.ContentListFragment">
+    <androidx.recyclerview.widget.RecyclerView
+      android:id="@+id/recyclerview"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_alignParentTop="true"
+      android:layout_marginStart="8dp"
+      android:layout_marginTop="4dp"
+      android:layout_marginEnd="8dp"
+      android:divider="@android:color/transparent"
+      android:dividerHeight="8dp"
+      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
```

Comment:
Note that you don't need to set the linear layout manager in the fragment presenter if you're specifying it here.

Reply:
Corrected.